sass/style-ec.scss

```diff
@@ -12,3 +12,12 @@
 @use "./components/media_iframe";
 @use "./components/expandable";
 @use "./components/contextual_navigation";
+
+// TODO: Remove after updating to ECL 3.9.1
+.ecl-list-illustration__image {
+  object-fit: contain;
+}
+// TODO: Remove after updating to ECL 3.9.1
+.ecl-list-illustration__image--square {
+  object-fit: cover;
+}
```

Comment:
The fix should be added also in the EU flavor:
![image](https://github.com/openeuropa/oe_theme/assets/9432036/33aa6e1c-d844-4504-9e5c-5257b3b1375a)